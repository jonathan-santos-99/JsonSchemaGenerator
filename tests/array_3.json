[
    1,
    "teste",
    {
        "obj": 2
    },
    [
        "array"
    ]
]