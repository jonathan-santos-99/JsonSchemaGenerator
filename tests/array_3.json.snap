0
eyIkc2NoZW1hIjoiaHR0cDovL2pzb24tc2NoZW1hLm9yZy9kcmFmdC0wNC9zY2hlbWEjIiwidHlwZSI6ImFycmF5IiwiaXRlbXMiOlt7InR5cGUiOiJpbnRlZ2VyIn0seyJ0eXBlIjoic3RyaW5nIn0seyJ0eXBlIjoib2JqZWN0IiwicHJvcGVydGllcyI6eyJvYmoiOnsidHlwZSI6ImludGVnZXIifX0sInJlcXVpcmVkIjpbIm9iaiJdfSx7InR5cGUiOiJhcnJheSIsIml0ZW1zIjpbeyJ0eXBlIjoic3RyaW5nIn1dfV19
